cinema/serializers.py

```diff
@@ -30,10 +30,26 @@ class Meta:
         fields = ("id", "name", "rows", "seats_in_row", "capacity")
 
 
+class MovieImageSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Movie
+        fields = ("id", "image")
+
+
 class MovieSerializer(serializers.ModelSerializer):
+    image = serializers.ImageField(read_only=True)
```

Comment:
You don't need to include image here - this serializer is used for creating new movie without image as per task requirements. You need to include image only on list and detail actions